cmd/erda-server/conf/i18n/issue-excel.yaml

```diff
@@ -2,7 +2,7 @@ zh:
   # Common
   Common: 通用字段
   ID: ID
-  IterationName: 迭代
+  s: 迭代
```

Comment:
The reference `s` is not very clear.

Reply:
This is a careless operation and has been fixed